docs/articles/samples/IntroCustomEnvironmentInfo.md

```diff
@@ -0,0 +1,38 @@
+---
+uid: BenchmarkDotNet.Samples.IntroCustomEnvironmentInfo
+---
+
+## Sample: IntroCustomEnvironmentInfo
+
+You can add any useful information about environment to the benchmark report. Just mark one or several static methods that return `string` or `IEnumerable<string>` by the [`[CustomEnvironmentInfo]`](xref:BenchmarkDotNet.Attributes.CustomEnvironmentInfoAttribute) attribute.
+
+### Source code
+
+[!code-csharp[IntroCustomEnvironmentInfo.cs](../../../samples/BenchmarkDotNet.Samples/IntroCustomEnvironmentInfo.cs)]
+
+### Output
+
+```
+BenchmarkDotNet=v0.11.1.20181013-develop, OS=Windows 10.0.17134.345 (1803/April2018Update/Redstone4)
```

Comment:
This documentation will be included in v0.11.2, so it's better to use `BenchmarkDotNet=v0.11.2` instead of `BenchmarkDotNet=v0.11.1.20181013-develop`